Wrapping/Generators/Python/Tests/extras.py

```diff
@@ -344,6 +344,11 @@ def custom_callback(name, progress):
     cast = image.astype(np.float32)
     assert cast.dtype == np.float32
 
+    npImage = (np.random.rand(12,8,3)*256).astype(np.uint8)
+    image = itk.GetImageFromArray(npImage, is_vector=True)
```

Comment:
To use a more consistent Pythonic style, please change `GetImageFromArray` to `image_from_array`.

Reply:
Done.